client/components/Editor/plugins/collaborative/document.js

```diff
@@ -65,13 +64,11 @@ export default (schema, props, collabDocPluginKey, localClientId) => {
 						/* https://firebase.google.com/docs/reference/js/firebase.database.Reference#transactionupdate:-function */
 						return undefined;
 					}
-					return {
-						id: uuidv4(), // Keyable Id
-						cId: localClientId, // Client Id
-						bId: ref.key.replace('branch-', ''), // Origin Branch Id
-						s: sendable.steps.map((step) => compressStepJSON(step.toJSON())),
-						t: firebaseTimestamp,
-					};
+					return createFirebaseChange(
```

Comment:
I don't love the word `keyable` - but it feels slightly more descriptive (at least internally) than `change`. In my mind a `firebaseChange` could be a cursor or metadata update. I think the underlying problem is that we likely just need a satisfying name for the thing we've been calling a `keyable`. That is, a name for the "steps + metadata" object.

Reply:
We invented the word `keyable` as shorthand for "`change` or `merge`", though — since this is specifically a `change`, it seems to under-specify what this object is. I am down to find a different name for changes and to propagate that through the codebase, though — I agree the name is vague enough to potentially refer to anything.

Reply:
Ah right. I forgot about the `merge` part of that. It seems like we may be able to deprecate the idea of storing `merges` as firebase objects that get concated into a list of changes now that Releases can work as pointers to a branch-key pair. I'm happy leaving this as is for now, and addressing it again once we figure out what the eventual firebase refactor looks like.